libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -379,6 +379,9 @@ singleton AP_Mission method get_item boolean uint16_t'skip_check mavlink_mission
 singleton AP_Mission method set_item boolean uint16_t'skip_check mavlink_mission_item_int_t
 singleton AP_Mission method clear boolean
 singleton AP_Mission method cmd_has_location boolean uint16_t'skip_check
+singleton AP_Mission method get_current_tag boolean uint16_t'Null
```

Comment:
i'd like to make this not auto-clear the flag, and instead return the number of NAV commands executed since the tag was started